bmc-reverse-proxy/pkg/bmc-reverse-proxy/main_test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+)
+
+func TestDirectInner(t *testing.T) {
+	cases := []struct {
+		host         string
```

Comment:
Give test cases unique names and run each as sub tests, as described below:
https://ieftimov.com/post/testing-in-go-subtests/